hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/RolloutHelper.java

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.jpa;
+
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.hawkbit.repository.exception.RolloutIllegalStateException;
+import org.eclipse.hawkbit.repository.exception.RolloutVerificationException;
+import org.eclipse.hawkbit.repository.model.Rollout;
+import org.eclipse.hawkbit.repository.model.RolloutGroup;
+import org.eclipse.hawkbit.repository.model.RolloutGroupConditions;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+final class RolloutHelper {
+    private RolloutHelper() {
+    }
+
+    static void verifyRolloutGroupConditions(final RolloutGroupConditions conditions) {
+        if (conditions.getSuccessCondition() == null) {
```

Comment:
You cloud also use org.springframework.util.Assert for your null checks
`Assert.notNull(conditions.getSuccessCondition(), "Rollout group is missing success condition");`
This would throw a IllegalArgumentException.